DigitalLearningSolutions.Data/Models/LearningResources/BulkLearningResourceReferences.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace DigitalLearningSolutions.Data.Models.LearningResources
+{
+    using DigitalLearningSolutions.Data.Models.External.LearningHubApiClient;
+
+    public class BulkLearningResourceReferences

```

Comment:
Now I'm seeing this in action, I'm starting to think is better just to use a tuple - probably shouldn't have expressed a preference on your HLD.
Less code, fewer similar-but-different model names, easier to see what a method's actually returning (as long as tuple components are named obvs). Whatcha think?

Reply:
Also applies to LearningResourceReferenceWithResourceDetails